deeprank_gnn/foutnet.py

```diff
@@ -82,29 +76,26 @@ def forward(self, x, edge_index):
         return alpha
 
     def __repr__(self):
-        return '{}({}, {})'.format(self.__class__.__name__,
-                                   self.in_channels,
-                                   self.out_channels)
+        return f"{self.__class__.__name__}({self.in_channels}, {self.out_channels})"
 
 
 class FoutNet(torch.nn.Module):
-
     def __init__(self, input_shape, output_shape=1, input_shape_edge=None):
```

Comment:
`input_shape_edge` argument is not used, but if I remove it the tests break (even if the argument doesn't seem used by the tests). Any idea about why does this happen?